Make.config

```diff
@@ -276,12 +276,12 @@ MACIOS_BINARIES_PATH=$(TOP)/external/macios-binaries
 
 MONO_PREFIX ?= /Library/Frameworks/Mono.framework/Versions/Current
 SYSTEM_MCS=$(MONO_PREFIX)/bin/mcs
-SYSTEM_CSC=$(MONO_PREFIX)/bin/csc
+SYSTEM_CSC=$(MONO_PREFIX)/bin/csc /features:strict
```

Comment:
This is wrong I believe. You want to add it more like this:

https://gist.github.com/408976c0b387172a2ae108766c4e2510 

where we put the other args.